packages/runner-shared/src/event-manager.js

```diff
@@ -582,12 +582,21 @@ export const eventManager = {
 
     Cypress.multiDomainCommunicator.on('after:screenshot', handleAfterScreenshot)
 
+    const crossOriginLogs = {}
+
     Cypress.multiDomainCommunicator.on('log:added', (attrs) => {
-      reporterBus.emit('reporter:log:add', attrs)
+      // Create a new local log representation of the cross origin log.
+      // It will be attached to the current command.
+      // We also keep a reference to it to update it in the future.
+      crossOriginLogs[attrs.id] = Cypress.log(attrs)
```

Comment:
Store the logs created by cross-origins. I could not find a good way to 'find' a log by id another way as i think they are spread out across many commands.